packages/contractkit/package.json

```diff
@@ -26,13 +26,14 @@
     "lint": "tslint -c tslint.json --project ."
   },
   "dependencies": {
-    "@0x/subproviders": "^5.0.0",
     "@celo/utils": "0.1.6-dev",
+    "@types/ethereumjs-util": "^5.2.0",
     "@types/debug": "^4.1.5",
     "bignumber.js": "^7.2.0",
     "cross-fetch": "3.0.4",
     "debug": "^4.1.1",
     "eth-lib": "^0.2.8",
+    "ethers": "4.0.0-beta.3",
```

Comment:
I'm not sure we want to add this dependency
there is ongoing work to minimize bundle sizes (ie stabilize version of web3 used across contractkit and mobile) and this bloats the package
I also don't think we want to encourage devs to use web3 and ethers

Reply:
The problem is that web3 is already using it. This is from the `yarn.lock`
```
web3-eth-abi@1.0.0-beta.37:
  version "1.0.0-beta.37"
  resolved "https://registry.yarnpkg.com/web3-eth-abi/-/web3-eth-abi-1.0.0-beta.37.tgz#55592fa9cd2427d9f0441d78f3b8d0c1359a2a24"
  integrity sha512-g9DKZGM2OqwKp/tX3W/yihcj7mQCtJ6CXyZXEIZfuDyRBED/iSEIFfieDOd+yo16sokLMig6FG7ADhhu+19hdA==
  dependencies:
    ethers "4.0.0-beta.1"
    underscore "1.8.3"
    web3-utils "1.0.0-beta.37"
```
Of course, I have to change the "4.0.0-beta.3" to "4.0.0-beta.1", but this dependency was already in use.
I've just added it because it was wrong to asume that we had it, just because we have the web3

Reply:
Ok, I can change this to use web3. We sill have the problem that web3 uses the ethers, but for now it's easier to use it from web3

From the version `2.0.0-alpha` they added its own implementation

Reply:
My two cents...

1. web3.js uses etherjs for abi coding. But it does some magic, so for now instead of using web3 abi coder, we should use ether.js directly.
2. We should use the exact same version that web3.js uses, so we don't have 2 different ether.js version around.

Reply:
hmm I didn't know web3 used ethers 🤷‍♂

Reply:
I checked and the function that we are using should work without any change from the web3 abi coder